hypertrace-trace-enricher/hypertrace-trace-enricher-api/src/test/java/org/hypertrace/traceenricher/trace/util/ApiTraceGraphDebug.java

```diff
@@ -12,237 +12,148 @@
-   Exit boundary Count: 0
-   ---
-   Api Node Index 5
-   Entry Event id: 3d4d74297f97c7f5, service_name: route
-   Events Count: 1
-   id: 3d4d74297f97c7f5, service_name: route
-   Exit boundary Count: 0
-   ---
-   Api Node Index 6
-   Entry Event id: 094dc03a19d69864, service_name: route
-   Events Count: 1
-   id: 094dc03a19d69864, service_name: route
-   Exit boundary Count: 0
-   ---
-   Api Node Index 7
-   Entry Event id: 3023972755b25ee7, service_name: route
-   Events Count: 1
-   id: 3023972755b25ee7, service_name: route
-   Exit boundary Count: 0
-   ---
-   Api Node Index 8
-   Entry Event id: 09e5688c857d918a, service_name: route
-   Events Count: 1
-   id: 09e5688c857d918a, service_name: route
-   Exit boundary Count: 0
-   ---
-   Api Node Index 9
-   Entry Event id: 3c91624cdb8d6a54, service_name: route
-   Events Count: 1
-   id: 3c91624cdb8d6a54, service_name: route
-   Exit boundary Count: 0
-   ---
-   Api Node Index 10
-   Entry Event id: 71cd8cdba7550d14, service_name: route
-   Events Count: 1
-   id: 71cd8cdba7550d14, service_name: route
-   Exit boundary Count: 0
-   ---
-   Api Node Index 11
-   Entry Event id: 161151419c2ce6b0, service_name: route
-   Events Count: 1
-   id: 161151419c2ce6b0, service_name: route
-   Exit boundary Count: 0
-   ---
-   Api Node Index 12
-   Entry Event id: 4301e4bbe15d1507, service_name: route
-   Events Count: 1
-   id: 4301e4bbe15d1507, service_name: route
-   Exit boundary Count: 0
-   ---
-   Edge Index 0
-   Api node index Src: 0, Tgt: 1
-   Event Id Src: 3e68fb176f4c336e, Tgt: 1bdfaf5ac2b6f0d3
-   ---
-   Edge Index 1
-   Api node index Src: 0, Tgt: 2
-   Event Id Src: 49d2f1013807f2ca, Tgt: 3aacfea7309a0605
-   ---
-   Edge Index 2
-   Api node index Src: 0, Tgt: 3
-   Event Id Src: 633537306bb06ab7, Tgt: 4df5bf1db4508049
-   ---
-   Edge Index 3
-   Api node index Src: 0, Tgt: 4
-   Event Id Src: 0e00b6161d108e4f, Tgt: 51093b5ee3ad58a8
-   ---
-   Edge Index 4
-   Api node index Src: 0, Tgt: 5
-   Event Id Src: 2d9c1a3926fbba6c, Tgt: 3d4d74297f97c7f5
-   ---
-   Edge Index 5
-   Api node index Src: 0, Tgt: 6
-   Event Id Src: 7185c5a864902e6d, Tgt: 094dc03a19d69864
-   ---
-   Edge Index 6
-   Api node index Src: 0, Tgt: 7
-   Event Id Src: 2db7e2b946f3412b, Tgt: 3023972755b25ee7
-   ---
-   Edge Index 7
-   Api node index Src: 0, Tgt: 8
-   Event Id Src: 3484acf9dfcf5f75, Tgt: 09e5688c857d918a
-   ---
-   Edge Index 8
-   Api node index Src: 0, Tgt: 9
-   Event Id Src: 6bea5062f49db48a, Tgt: 3c91624cdb8d6a54
-   ---
-   Edge Index 9
-   Api node index Src: 0, Tgt: 10
-   Event Id Src: 25ba455250f612c0, Tgt: 71cd8cdba7550d14
-   ---
-   Edge Index 10
-   Api node index Src: 0, Tgt: 11
-   Event Id Src: 2e35bf7d6fb69be6, Tgt: 161151419c2ce6b0
-   ---
-   Edge Index 11
-   Api node index Src: 0, Tgt: 12
-   Event Id Src: 24ebcce46fa077b9, Tgt: 4301e4bbe15d1507
-   ---
+   * ApiTraceGraph for Hotrod trace --- Api Node Index 0 Entry Event id: 383acc79e53b84da,
```

Comment:
Huh. It does not know not to reformat this.

Reply:
More like we don't know how to format it :) It's a javadoc-style comment so formatting is based on how it would appear in javadocs, not in code. If this was meant for the reader, it should have used `//` style.

Reply:
(not that we shouldn't fix it - but we should revert and swap the comment style)

Reply:
fixed. Threw a `<pre>` block around it